python/hawkey/reldep-py.c

```diff
@@ -135,10 +135,6 @@ reldep_init(_ReldepObject *self, PyObject *args, PyObject *kwds)
     g_free(name);
     g_free(evr);
     Py_XDECREF(tmp_py_str);
-    if (self->reldep == NULL) {
```

Comment:
But it was possible to do that before, but according to description in commit message it will be impossible to do it after patch.